### ISSUE TITLE
Screen shot documentation seems incomplete

### ISSUE THREAD

**Issue by talhahasanzia** [mobile-dev-inc/maestro-docs#37]
The screen shot documentation is as follows:

# takeScreenshot

`takeScreenshot` saves a screenshot in a PNG file:

```yaml
- takeScreenshot: MainScreen    # will be stored as a MainScreen.png
```

[Reference](https://github.com/mobile-dev-inc/maestro-docs/blob/main/api-reference/commands/takescreenshot.md)

It would be great if we can somehow mention where the screenshot will be saved and if we can change output directory like we can do in test reports.

**Comment by bartekpacia** [comment]
Hi @talhahasanzia, I agree we could do a better job here.

Contributions are very welcome!

**Comment by niklaskroe** [comment]
should be closed with merging of PR #64

**Comment by bartekpacia** [comment]
closed by #64
